modules/exploits/windows/http/file_sharing_wizard_seh.rb

```diff
@@ -0,0 +1,57 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Seh
+
+  def initialize(info = {})
+    super update_info(info,
+                      'Name' => 'File Sharing Wizard 1.5.0 - POST SEH Overflow',
+                      'Description' => %q(
+        This module exploits a SEH based buffer overflow in an HTTP POST parameter in File Sharing Wizard 1.5.0
+      ),
+                      'Author' => [
+                        'x00pwn', # Original exploit
+                        'Dean Welch <dean_welch[at]rapid7.com>' # Module
+                      ],
+                      'License'        => MSF_LICENSE,
+                      'References'     =>
+                          [
+                            %w(CVE 2019-16724),
```

Comment:
`%w[]` is preferred syntax.

Reply:
so many ways to do the same thing in ruby, is there a functional difference or just styling?

Reply:
Styling that suggests function: arrays are formed with `[` and `]`, so it's a visual cue.